plans/session-recording.fmf

```diff
@@ -1,9 +1,18 @@
-summary:
-  Tests using pre-recorded responses from external services.
-prepare:
-  how: ansible
-  playbooks:
-  - files/packit-testing-farm-prepare-session-recording.yaml
+summary: Session recording testcases
+discover:
+  how: fmf
+  filter: tier:2
 execute:
-  script:
-  - make check TESTS_TARGET=tests_recording
+  how: tmt
+adjust:
+  when: "distro <= rhel-8 or distro <= centos-8"
+  prepare:
+    - how: install
+      name: ensure_copr_plugin
+      order: 10
+      package:
+        - dnf-plugins-core
+    - how: shell
+      name: requre_copr
+      order: 20
+      script: 'dnf -y copr enable jscotka/requre'
```

Comment:
Note: Once the new tmt is released and deployed in the pipeline the following should be enough:

```yaml
prepare:
    how: install
    copr: jscotka/requre
```

Reply:
> Note: Once the new tmt is released and deployed in the pipeline the following should be enough:
> 
> ```yaml
> prepare:
>     how: install
>     copr: jscotka/requre
> ```

@psss does that mean that this enable copr repo and packages could be part of L1 metadata? in that case it could be great.

Reply:
No, currently this is only L2. And psss/tmt/issues/647 allows to enable copr repo without specifying a package.

Reply:
That is great news.

Reply:
@jscotka Why don't you use Copr repo made by Packit:

* https://copr.fedorainfracloud.org/coprs/packit/packit-requre-main/

Reply:
there is missing rhel/epel builds, what is the reason why I'm using COPR, in fedora there is requre already packed.

> @jscotka Why don't you use Copr repo made by Packit:
> 
>     * https://copr.fedorainfracloud.org/coprs/packit/packit-requre-main/

there is missing rhel/epel builds, what is the reason why I'm using COPR, in fedora there is requre already packed.

Reply:
fixed by https://github.com/packit/requre/pull/174